build.gradle.kts

```diff
@@ -75,7 +88,14 @@ kotlin {
         val nativeTest by creating {
             dependsOn(commonTest)
         }
+        val jsMain by getting {
 
+        }
```

Comment:
you can remove this one. it's added by the plugin, and nothing is added here. if it were absolutely needed, this would need to be `by creating`